Description: fix double definition of BYTE_ORDER on hppa
Author: Helge Deller <deller@gmx.de>
Bug-Debian: https://bugs.debian.org/808990
Updated: 2024-03-21

--- a/nss/lib/dbm/include/mcom_db.h
+++ b/nss/lib/dbm/include/mcom_db.h
@@ -110,7 +110,7 @@ typedef PRUint32 uint32;
 #endif /* !BYTE_ORDER */
 #endif /* __sun */
 
-#if defined(__hpux) || defined(__hppa)
+#if defined(__hpux)
 #ifndef BYTE_ORDER
 #define BYTE_ORDER BIG_ENDIAN
 #define BIG_ENDIAN 4321
